issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraProject.java

```diff
@@ -36,6 +36,7 @@ public class JiraProject implements IssueProject {
 
     public static final String RESOLVED_IN_BUILD = "customfield_10006";
     public static final String SUBCOMPONENT = "customfield_10008";
+    public static final String JEP_NUMBER = "customfield_10701";
```

Comment:
Don't think this is needed anymore?

Reply:
Why? It is used in many test cases and the convention is putting such field name at the `JiraProjec.java`.

Reply:
Aha, I did not notice that. I only looked at that particular class. Nevermind, then.